### ISSUE TITLE
"Author" in author string of some names

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#371]
https://www.catalogueoflife.org/data/search?q=author

**Comment by yroskov** [comment]
Examples from WoRMS:

![image](https://user-images.githubusercontent.com/28574252/145606654-b2b7dfc9-76cf-41b8-bd13-1c43e82d83d6.png)

![image](https://user-images.githubusercontent.com/28574252/145606715-757cbc16-6e48-48e6-b176-fe8f9e9f0a89.png)

**Comment by yroskov** [comment]
Example from Kew database:

![image](https://user-images.githubusercontent.com/28574252/145606869-8f15607b-29cd-4608-8f25-a9c54f6d2a85.png)

![image](https://user-images.githubusercontent.com/28574252/145606999-f1f084e5-da0b-494c-8585-7ba05af27575.png)

**Comment by yroskov** [comment]
Examples from ILDIS:

![image](https://user-images.githubusercontent.com/28574252/145609270-63330d94-2459-4604-b49a-2ae65c8c84c1.png)

![image](https://user-images.githubusercontent.com/28574252/145609338-9a8bf00b-bf45-463a-a966-eaec5484bffe.png)

Corrected: names marked as "provisionally accepted"

**Comment by yroskov** [comment]
Example from World Plants:

![image](https://user-images.githubusercontent.com/28574252/145610063-2439d518-e5ea-4e09-b43e-e61a838b39b3.png)

"Author not stated" in IPNI as well.

**Comment by yroskov** [comment]
I do not regard this as a problem. The CoL is publishing these data correctly, as they appear in source databases. As soon as authorstrings will be resolved in WoRMS, ILDIS and World Plants, data will appear in the CoL.

In the case of Kew checklist, the authorstring included a comment - the parser problem. However, I always happy to see authorstrings in full, as they presented by the data provider.

**Comment by TonyRees** [comment]
I can comment on this from the WoRMS + IRMNG perspective ... let us say you have a name _Andromeda_ Smith (I just made it up, probably does not exist...) Then you have an instance of _Andromeda_ as a separate taxon somewhere else in the taxonomic tree, authored by someone else (i.e., a homonym), the source from which the name does not specify the authorship, but you wish to enter it as a separate taxon anyway for whatever reason (e.g. so that its species can be included or whatever). Now, the data entry software created for WoRMS, also now used for IRMNG, considers "Andromeda Smith" and "Andromeda" (without authorship) as the same name and will not let you enter a second instance, so the second one has to be distinguished by some different content in the "authority" field ... so for IRMNG I might enter it as e.g. "Andromeda [author]", or "Andromeda [author?]", or similar... just a workaround for convenience, until the correct authorship can be determined. Or if I am aware that the second one is a published misspelling, but wish to enter it anyway, I might put "Andromeda [misspelling]", thus [misspelling] will then end up in the authorship field - as I say, something is needed there so that the system will accept the name via the data entry form. (Of course this only applies to homonyms, names that do not already occur in the system can be entered without authorship if needed, as has happened in some small % of cases in the past).

However in other cases that you give, such as "not accepted by author", that is really a comment and should be parsed out separately...

Cheers - Tony

**Comment by mdoering** [comment]
> I do not regard this as a problem. The CoL is publishing these data correctly, as they appear in source databases. As soon as authorstrings will be resolved in WoRMS, ILDIS and World Plants, data will appear in the CoL.

Yes, but it is the role of COL to do editorial work on top of the supplied data. We do this in various places, so I would clearly expect that to happen also in those authorstring cases above. An editorial decision would help to remove these problem!

> In the case of Kew checklist, the authorstring included a comment - the parser problem. However, I always happy to see authorstrings in full, as they presented by the data provider.

Parser or editorial problem. It is impossible to parse out all problems automatically, that's why we have editorial decisions oin the software to fix such things!



> I do not regard this as a problem. The CoL is publishing these data correctly, as they appear in source databases. As soon as authorstrings will be resolved in WoRMS, ILDIS and World Plants, data will appear in the CoL.
> 
> In the case of Kew checklist, the authorstring included a comment - the parser problem. However, I always happy to see authorstrings in full, as they presented by the data provider.

**Comment by mdoering** [comment]
@yroskov or should we take this to the taxonomic group and ask advice? I usually see these problem from a user's perspective and those cases are not expected

**Comment by yroskov** [comment]
What TG or CoL can do in such cases? It is entirely on GSDs and their taxonomists. 

As many changes will be applied on the side of CoL (as well as in GBIF) as messier data become soon.

**Comment by dhobern** [comment]
It is important that we understand the structure of the data we received and that we do all we can to ensure that users of these data - particularly those who rely on APIs and structured data exports - do not have to carry out needless extra processing to deal with issues that we can address once. 

It seems to me that we are dealing here with a verbatim version as received from the source and that we then need to consider the appropriate way to handle and represent this in different contexts. It may be that a human readable view should just present whatever the source provided, maybe with an interpretative note. For data exports/downloads/APIs, we should avoid including avoidable noise in these channels. Computer software should be able to rely on direct string comparisons rather than having to preprocess the COL data to get it clean enough to compare.

So my view is that we need to have a clean version of these fields (without the extra author) in any context where there is an expectation of machines using the data.

**Comment by yroskov** [comment]
Do you believe, machines cannot use a value **(Author)**?

**Comment by yroskov** [comment]
Actually, what is a big difference between Pterophorus xanthodactylus **auct. [nec Treitschke, 1833]** and Nymphaea gigantea f. hudsonii **(Author)** ?

**Comment by yroskov** [comment]
Corrections should be done in GSDs, in the source checklists, not in the CoL or GBIF. To do that, we need to establish IT  infrastructure for a feedback to GSD authors, and their good will do a work.

**Comment by dhobern** [comment]
I agree that the changes need to be in the GSDs but there is a whole spectrum of tiny to major things that are easily intercepted at the COL level and prevented from propagating further. These should be logged for subsequent fixing but may be very hard to correct in whatever system the GSD uses or else may require more work than they can handle in any reasonable timeframe.

**Comment by bart-v** [comment]
WoRMS cases fixed. Available in next COL export 2022-06-01

- Amphicteis brivispinis
- Lumbrineris capensis

Bad habit of some editors.
We'll spank them :)
